libs/openFrameworksCompiled/project/osx/CoreOF.xcconfig

```diff
@@ -1,3 +1,19 @@
+ARCHS = $(ARCHS_STANDARD_32_64_BIT)	// this allows you to select target architecture in dropdown menu
+VALID_ARCHS = x86_64 i386
+
+ONLY_ACTIVE_ARCH = YES      // this makes sure not to build an universal binary
+
+//// define which C++ library is required to build, based on target architecture
+
+CLANG_CXX_LIBRARY[arch=i386] = libstdc++
+CLANG_CXX_LANGUAGE_STANDARD[arch=i386] = gnu++98
+
+CLANG_CXX_LIBRARY[arch=x86_64] = libc++
+CLANG_CXX_LANGUAGE_STANDARD[arch=x86_64] = c++11
+
+MACOSX_DEPLOYMENT_TARGET = 10.7		// libstdc++ is only available in 10.7+
```

Comment:
Interesting, so we *cannot* support Mac OS X 10.6? That is perfectly fine with me as I was advocating we drop 10.6 and 10.7, see #3507 and #3543 :wink: